class.ext_update.php

```diff
@@ -61,4 +78,82 @@ public function main()
 		return '<pre>' . $output . '</pre>' . PHP_EOL .
 			sprintf( "Setup process lasted %1\$f sec</br>\n", (microtime( true ) - $exectimeStart) );
 	}
+
+	/**
+	 * Check the environment.
+	 *
+	 * TYPO3 9.5 and MySql 5.6 and below may make problems. This was introduced
+	 * with #80398.
+	 *
+	 * @return bool
+	 */
+	protected function checkEnvironment(): bool
+	{
+		if ( version_compare( TYPO3_version, '9.5', '<' ) )
```

Comment:
The problem can also exist with TYPO3 v9 because TYPO3 doesn't require long indexes itself.

Reply:
Can also happen with TYPO3 9.5 and MySQL <5.7